### ISSUE TITLE
[1.7] Laser sound missing

### ISSUE THREAD

**Issue by LemADEC** [BuiltBrokenModding/ICBM-Classic#79]
As of ICBM-classic-1.7.10-2.16.3b212, sounds.json is referring to invalid laser1.ogg file:
```
[11:21:30] [Client thread/WARN]: File icbmclassic:sounds/laser1.ogg does not exist, cannot add it to event icbmclassic:laser
```

**Comment by Jkosio23** [comment]
Missing again in icbm-classic-1.12.2.3.0.0b29

**Comment by DarkGuardsman** [comment]
It was never fixed, closed the ticket as I was not going to fix it for 1.7
